### PR TITLE
docs(connectors): add docs for secure TLS connexion

### PR DESCRIPTION
**Pull request by mathias-vandaele**
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by sbuettner — DISMISSED** [review]

File: docs/self-managed/connectors-deployment/connectors-configuration.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ If you are connecting a local Connector runtime to a SaaS cluster, you may want
 
 Zeebe:
 
+### Secure connection
+
+| Environment variable                                | Purpose                                                                        |
+| :-------------------------------------------------- | :----------------------------------------------------------------------------- |
+| `CAMUNDA_CLIENT_ZEEBE_BASEURL` (required)           | The base url of the zeebe broker (https)                                       |
```
Comment (ev-codes) [inline docs/self-managed/connectors-deployment/connectors-configuration.md:57]:
```suggestion
| `CAMUNDA_CLIENT_ZEEBE_BASEURL` (required)           | The base URL of the Zeebe broker (HTTPS)                                       |
```

File: docs/self-managed/connectors-deployment/connectors-configuration.md [hunk 2]
```diff
@@ -50,6 +50,13 @@ If you are connecting a local Connector runtime to a SaaS cluster, you may want
...
+| `CAMUNDA_CLIENT_ZEEBE_CACERTIFICATEPATH` (optional) | The file location of the certificate to be used to connect to the zeebe broker |
```
Comment (ev-codes) [inline docs/self-managed/connectors-deployment/connectors-configuration.md:58]:
```suggestion
| `CAMUNDA_CLIENT_ZEEBE_CACERTIFICATEPATH` (optional) | The file location of the certificate to be used to connect to the Zeebe broker |
```

**Review by ev-codes — CHANGES_REQUESTED** [review]

**Review by ev-codes — APPROVED** [review]

**Comment by ev-codes** [comment]
@mathias-vandaele thanks, looks good!

I went ahead and capitalized terms that needed it (e.g., Zeebe). In public-facing docs, it's best to follow convention for easier reading 😄

**Comment by akeller** [comment]
Hey there! Please be sure to fill out the PR description and add labels. This helps the DevEx team, specifically the technical writers, know how and when to review and release the docs. Also, be sure to link back to Product Hub epics or team board issues so we have more context about this change, as we often try to do some validation (is this change properly backported, are there other areas that need to be changed as well, etc.).

**Pull request merged**